packages/uikit/src/molecules/Navbar/CollapseIcon.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled, { css, keyframes } from '../../lib/styledComponents';
+
+interface Props {
+  onClick: () => void;
+  open?: boolean;
+}
+
+const time = '0.15s';
+const delay = '.25s';
+const cubic = 'cubic-bezier(.25, .73, .85, .56)';
+const height = 30;
+
+const animate = keyframes`
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(45deg);
+  }
+`;
+
+const animateUp = keyframes`
+  0% {
+    transform: rotate(0deg);
+  }
+100% {
+    transform: rotate(-45deg);
+}
+`;
+
+const Line = styled.span`
```

Comment:
love it, but you're missing one 😉 - I'd be so angry if I ordered a hamburger without the actually patty

Reply:
It's a low-carb one